.github/workflows/helm.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
 
       - uses: actions/setup-go@v3
         with:
-          go-version: '1.22.3'
+          go-version-file: 'go.mod'
```

Comment:
Nice: https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file